<ui version="4.0" >
 <class>MailEditorDialog</class>
 <widget class="QMainWindow" name="MailEditorDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Nuntius Leo - Editor</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources.qrc" >
    <normaloff>:/img/images/mail_new.png</normaloff>:/img/images/mail_new.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>55</y>
     <width>614</width>
     <height>410</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>From:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QLineEdit" name="from" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>To:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QLineEdit" name="to" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="label_4" >
      <property name="text" >
       <string>Cc:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QLineEdit" name="cc" />
    </item>
    <item row="3" column="0" >
     <widget class="QLabel" name="label_5" >
      <property name="text" >
       <string>Bcc:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" >
     <widget class="QLineEdit" name="bcc" />
    </item>
    <item row="4" column="0" >
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>Subject:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" >
     <widget class="QLineEdit" name="subject" >
      <property name="text" >
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2" >
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTextEdit" name="text" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>4</verstretch>
        </sizepolicy>
       </property>
       <property name="html" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
       </property>
      </widget>
      <widget class="QTreeWidget" name="attachments" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <column>
        <property name="text" >
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Size</string>
        </property>
       </column>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>614</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionSendMail" />
    <addaction name="actionSave" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menu_Insert" >
    <property name="title" >
     <string>&amp;Insert</string>
    </property>
    <addaction name="actionAddAttachment" />
   </widget>
   <widget class="QMenu" name="menu_View" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_Insert" />
   <addaction name="menu_View" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>465</y>
     <width>614</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="fileToolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>25</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionSendMail" />
   <addaction name="separator" />
   <addaction name="actionSave" />
   <addaction name="separator" />
   <addaction name="actionAddAttachment" />
  </widget>
  <widget class="QToolBar" name="textEditToolBar" >
   <property name="geometry" >
    <rect>
     <x>99</x>
     <y>25</y>
     <width>515</width>
     <height>30</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>Editor Toolbar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionToggleHtml" />
   <addaction name="separator" />
   <addaction name="actionBold" />
   <addaction name="actionItalic" />
   <addaction name="actionUnderlined" />
   <addaction name="separator" />
   <addaction name="actionAlignmentLeft" />
   <addaction name="actionAlignmentCenter" />
   <addaction name="actionAlignmentRight" />
   <addaction name="actionAlignmentJustified" />
  </widget>
  <action name="actionSendMail" >
   <property name="icon" >
    <iconset resource="../resources.qrc" >
     <normaloff>:/img/images/mail_send.png</normaloff>:/img/images/mail_send.png</iconset>
   </property>
   <property name="text" >
    <string>Send Mail</string>
   </property>
  </action>
  <action name="actionBold" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources.qrc" >
     <normaloff>:/img/images/text_bold.png</normaloff>:/img/images/text_bold.png</iconset>
   </property>
   <property name="text" >
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources.qrc" >
     <normaloff>:/img/images/text_italic.png</normaloff>:/img/images/text_italic.png</iconset>
   </property>
   <property name="text" >
    <string>Italic</string>
   </property>
  </action>
  <action name="actionUnderlined" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources.qrc" >
     <normaloff>:/img/images/text_under.png</normaloff>:/img/images/text_under.png</iconset>
   </property>
   <property name="text" >
    <string>Underlined</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="icon" >
    <iconset resource="../resources.qrc" >
     <normaloff>:/img/images/exit.png</normaloff>:/img/images/exit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAlignmentLeft" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources.qrc" >
     <normaloff>:/img/images/text_left.png</normaloff>:/img/images/text_left.png</iconset>
   </property>
   <property name="text" >
    <string>Left Align</string>
   </property>
  </action>
  <action name="actionAlignmentCenter" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources.qrc" >
     <normaloff>:/img/images/text_center.png</normaloff>:/img/images/text_center.png</iconset>
   </property>
   <property name="text" >
    <string>Center Align</string>
   </property>
  </action>
  <action name="actionAlignmentRight" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources.qrc" >
     <normaloff>:/img/images/text_right.png</normaloff>:/img/images/text_right.png</iconset>
   </property>
   <property name="text" >
    <string>Right Align</string>
   </property>
  </action>
  <action name="actionAlignmentJustified" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources.qrc" >
     <normaloff>:/img/images/text_block.png</normaloff>:/img/images/text_block.png</iconset>
   </property>
   <property name="text" >
    <string>Block Align</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="icon" >
    <iconset resource="../resources.qrc" >
     <normaloff>:/img/images/3floppy_unmount.png</normaloff>:/img/images/3floppy_unmount.png</iconset>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
  <action name="actionAddAttachment" >
   <property name="icon" >
    <iconset resource="../resources.qrc" >
     <normaloff>:/img/images/attach.png</normaloff>:/img/images/attach.png</iconset>
   </property>
   <property name="text" >
    <string>Add Attachment...</string>
   </property>
  </action>
  <action name="actionToggleHtml" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources.qrc" >
     <normaloff>:/img/images/html.png</normaloff>:/img/images/html.png</iconset>
   </property>
   <property name="text" >
    <string>Toggle Html</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionToggleHtml</sender>
   <signal>triggered(bool)</signal>
   <receiver>actionBold</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggleHtml</sender>
   <signal>triggered(bool)</signal>
   <receiver>actionItalic</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggleHtml</sender>
   <signal>triggered(bool)</signal>
   <receiver>actionUnderlined</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
